src/ceph-daemon/ceph-daemon

```diff
@@ -280,6 +280,47 @@ def make_log_dir(fsid, uid=None, gid=None):
     makedirs(log_dir, uid, gid, LOG_DIR_MODE)
     return log_dir
 
+def copy_file(src, dst, uid=None, gid=None):
+    """
+    Copy a file from src to dst
+    """
+    # type: (str, int, int) -> str
```

Comment:
```
ceph-daemon:287: error: misplaced type annotation
```

Reply:
oops .. there were two instances of this .. 

The type hint has been placed in the correct location, as well as adding the missing `str` param 

```suggestion
    # type: (str, str, int, int) -> str
```